mobile/mysterium/entrypoint.go

```diff
@@ -298,16 +319,19 @@ func NewNode(appPath string, options *MobileNodeOptions) (*MobileNode, error) {
 				ChannelImplAddress: options.ChannelImplementationSCAddress,
 				MystAddress:        options.MystSCAddress,
 				ChainID:            options.Chain1ID,
+				KnownHermeses:      config.GetStringSlice(config.FlagChain1KnownHermeses),
 			},
 			Chain2: metadata.ChainDefinition{
 				RegistryAddress:    options.RegistrySCAddress,
 				HermesID:           options.HermesSCAddress,
 				ChannelImplAddress: options.ChannelImplementationSCAddress,
 				MystAddress:        options.MystSCAddress,
 				ChainID:            options.Chain2ID,
+				KnownHermeses:      config.GetStringSlice(config.FlagChain2KnownHermeses),
 			},
 		},
-		Consumer:        true,
+
+		Consumer:        false,
```

Comment:
@Zensey if we merge this, will it break our current mobile app? 
Feels like it should be in a separate branch to keep our mobile builds alive.

Reply:
@soffokl 
as long as it was decided to make a separate app for provider, I'll make a separate function